language/move-lang/stdlib/modules/libra_time.move

```diff
@@ -0,0 +1,42 @@
+address 0x0:
+
+module LibraTimestamp {
+    use 0x0::Transaction;
+
+    // A singleton resource holding the current Unix time in microseconds
+    resource struct CurrentTimeMicroseconds {
+        microseconds: u64,
+    }
+
+    // Initialize the global wall clock time resource.
+    public fun initialize_timer() {
+
+        // Only callable by the Association address
+        Transaction::assert(Transaction::sender() == 0xA550C18, 1);
+
+        // TODO: Should the initialized value be passed in to genesis?
+        let timer = CurrentTimeMicroseconds {microseconds: 0};
+        move_to_sender<CurrentTimeMicroseconds>(timer);
+    }
+
+    // Update the wall clock time by consensus. Requires VM privilege and will be invoked during block prologue.
+    public fun update_global_time(proposer: address, timestamp: u64) acquires CurrentTimeMicroseconds {
+        // Can only be invoked by LibraVM privilege.
+        // Fixme: uncomment when after adding support in functional test.
```

Comment:
@tzakian is this fixme from you?

Reply:
No. This is an existing FIXME from IIRC @runtian-zhou

Reply:
We should probably look into fixing this ASAP
@runtian-zhou @vgao1996

Reply:
It was basically because we are calling into this function directly in functional test to test the behavior of block prologue, and adding this assertion will fail those tests. I'll add support for functional test soon so that we can add the assertions back.